make-go.bash

```diff
@@ -60,6 +64,11 @@ mkdir -p "$GOROOT_FINAL"
 
 pushd ext/golang/go/src
 
+for PATCH in ../../../../patches/golang/all/*.patch; do
+    echo "Applying $PATCH"
+    git apply $PATCH
+done
```

Comment:
The patch isn't applied, not quite sure why. Maybe it's because the file is in a different git repo. It might be better to use `patch`.